### ISSUE TITLE
Pickup broken blocks

### ISSUE THREAD

**Issue by Melodi17** [cabaletta/baritone#4151]
## Describe your suggestion
**For situations where you need extra blocks, and using goto, it could ensure to pick up blocks it breaks along the way, and also when the #sel fill/replace commands are used.**

## Settings
**This would be a setting that would apply to all the other commands around block breaking, mine, goto (if allowBreak is on), #sel fill/replace, fill, etc. Possibly there could be specific blocks set? e.g only try to collect blocks specifically targeted?**

## Context
Describe how your suggestion would improve Baritone, or the reason behind it being added.
**This would provide a lot of convenience to the users (and me), and enable a lot more things, such as being able to empty out an area but collect the blocks instead of letting them despawn.**

## Final checklist
- [x] I know how to properly use check boxes 
- [x] I have not used any OwO's or UwU's in this issue.

**Comment by RealmKebab** [comment]
this is already a thing?

**Comment by ZacSharp** [comment]
No, it's not. Baritone only picks up items while mining or farming.

**Comment by MendingNights** [comment]
Might not be practical for traveling or pathing, but when mining or terraforming, I use Alays to collect dropped blocks.
